cnc_ctrl_v1/Report.cpp

```diff
@@ -236,7 +236,20 @@ void  returnPoz(){
     static unsigned long lastRan = millis();
     
     if (millis() - lastRan > POSITIONTIMEOUT){
+        char tmpbuf[80], buflen=0;
+        memset(tmpbuf, 0, sizeof(tmpbuf));
         
+        for (int i=0;i<3;i++){
+            if(axes[i].stalled > 5){
+                buflen+=snprintf(tmpbuf+buflen, sizeof(tmpbuf)-buflen, "%c %s ", axes[i].name(), (axes[i].moved > 5) ? F("STALL") : F("UNHK "));
+            }
+        }
+        
+        if(buflen>0){
+            Serial.print(F("Message: "));
```

Comment:
My preference would be to use the reportAlarmMessage or reportFeedbackMessage functions instead of this.  I kinda feel like this is an alarm instance.  

I think these functions will give us more flexibility in the future (other languages, only reporting error numbers to save program space).

Reply:
Sounds like a good idea.  I borrowed half of that idea mainly because I want to see what the debug value is in case of a stall...